src/random_test.cpp

```diff
@@ -976,7 +976,8 @@ void Table::Analyze(Thd1 *thd) {
 
 void Table::Truncate(Thd1 *thd) {
   /* 99% truncate the some partition */
-  if (type == PARTITION && rand_int(100) > 98) {
```

Comment:
existing problem. it should be 1 instead of 98. based on the above comment. please fix it as part of this patch.

Reply:
Ok. Do we need to retain the OR condition I have added ?